src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -237,10 +237,18 @@ public static void GetPathRoot_Unix()
             Assert.False(Path.IsPathRooted(uncPath));
             Assert.Equal(string.Empty, Path.GetPathRoot(uncPath));
         }
-        
+
+        [Fact]
+        public static void IsPathRooted()
+        {
+            Assert.False(Path.IsPathRooted(null));
+        }
+
         // Testing invalid drive letters !(a-zA-Z)
         [PlatformSpecific(TestPlatforms.Windows)]
         [Theory]
+        [InlineData("")]
+        [InlineData(" ")]
```

Comment:
This two cases return false as well on Desktop and this tests is skipped on Net Framework. Could we instead move this test cases to `IsPathRooted()` test that you are adding above? Make it a theory and pass as inline data:
```cs
[InlineData("")]
[InlineData(" ")]
[InlineData(null)]
```